### DISCUSSION TITLE
Pull Docker images in parallel

### DISCUSSION THREAD

**Discussion by mvorisek** [community/community#9970] [Actions]
Pull operations contribute significantly to the total execution time.

Repos with heavy CI testing like https://github.com/atk4/data have a lot of services (different Docker Images) and starting the services takes up too several minutes.

This is a feature request to improve the Github Actions by pulling the Docker images in parallel and reduce the total CI pipeline time significantly.

**Comment by ethomson** [comment]
Thanks @mvorisek - this seems like a useful improvement and I've added it to the backlog.

**Reply by mvorisek** [reply]
@ethomson can you please update me about this feature requests status?

**Reply by mvorisek** [reply]
Can this be implemented please? It will massively improve workflow startup time when many services need to be booted. Thank you.

**Comment by mvorisek** [comment]
duplicate of https://github.com/actions/runner/issues/1376
